tests/pipelines/stable_diffusion/test_stable_diffusion_adapter.py

```diff
@@ -171,6 +175,36 @@ def test_inference_batch_single_identical(self):
         self._test_inference_batch_single_identical(expected_max_diff=2e-3)
 
 
+    def test_image_dimensions(self):
```

Comment:
```suggestion
    def test_multiple_image_dimensions(self):
```

Indicates the objective better, I guess?

Reply:
✅ Renamed